### PR TITLE
UI & Logo Updates and Bug fixes

### PR DESCRIPTION
**Pull request by thetechie7**
1. Updated Featuring Keynote Speakers Cards and Background.

2. Fixed FAQ Expand all button, now there is way to close them all at once too and also a small transition animation for the chevron.

3. Updated Hero App button styling

4. Updated Font to Roboto

5. Updated Logos 

6. Fixed App Header Logo sizing and added cursor pointer to the menu button.

### CODE REVIEW

**Review by NickBurnett — APPROVED** [review]

File: components/faq.tsx [hunk 1]
```diff
@@ -54,13 +58,23 @@ export default function FaqPage({ fetchedFaqs }: { fetchedFaqs: AnsweredQuestion
           <div className="flex flex-row items-center gap-x-2">
             <button
               onClick={() => {
-                expandAll();
+                if (disclosuresStatus.every((status) => status)) {
+                  closeAll();
+                } else {
+                  expandAll();
+                }
```
Comment (MichaelZhao21) [inline components/faq.tsx:65]: Extract into named function instead of anonymous lambda function

File: components/faq.tsx [hunk 2]
```diff
@@ -54,13 +58,23 @@ export default function FaqPage({ fetchedFaqs }: { fetchedFaqs: AnsweredQuestion
...
+              {disclosuresStatus.every((status) => status) ? 'Close All' : 'Expand All'}
             </button>
-            <ChevronDownIcon className="w-5 h-5" />
+            <ChevronDownIcon
+              className={`${
+                disclosuresStatus.every((status) => status)
+                  ? 'transform rotate-180 transition duration-500 ease-in-out'
+                  : 'transition duration-500 ease-in-out'
+              } w-5 h-5`}
+            />
```
Comment (MichaelZhao21) [inline components/faq.tsx:77]: same repeated code above -- maybe extract this into its own component

File: pages/index.tsx [hunk 3]
```diff
@@ -256,27 +256,23 @@ export default function Home(props: {
...
+              <KeynoteSpeaker
+                key={idx}
+                name={name}
+                description={description}
+                cardColor={colorSchemes[idx % 3]}
+                imageLink={fileName}
+              />
+            ))}
+          </div>
+          {/* row 2
```
Comment (MichaelZhao21) [inline pages/index.tsx:275]: this is prob fine but will almost 100% create merge conflicts with #129

**Review by MichaelZhao21 — CHANGES_REQUESTED** [review]
Just as a general note, you shouldn't combine changes across multiple unrelated components into a single PR

File: pages/index.tsx (hunk 3 above)
Comment (thetechie7) [reply pages/index.tsx:275]: so you want me to remove it?

File: components/faq.tsx (hunk 1 above)
Comment (thetechie7) [reply components/faq.tsx:65]: done, made it FaqExpandCloseButton component

File: components/KeynoteSpeaker.tsx [hunk 4]
```diff
@@ -25,25 +25,26 @@ export default function KeynoteSpeaker(props) {
...
           />
         )}
       </div>
-      <div style={{ backgroundColor: props.cardColor.dark }} className="w-3/4 p-2 rounded-r-md">
-        <h1 className="text-lg font-bold"> {props.name}</h1>
-        <div className="text-xs">{props.description}</div>
+      <div className=" flex-col items-center justify-center  sm:w-60 sm:h-80 w-40 h-60  absolute  translate-y-60 group-hover:translate-y-0 transition duration-500 ease-in-out overflow-hidden ">
+        <div className="rounded-b-sm bg-indigo-300  p-2 font-bold ">
+          <h1 className="text-lg "> {props.name}</h1>
+          {/* show description on hover  by sliding up */}
```
[the remark this answers is not in the document]
Comment (thetechie7) [reply components/KeynoteSpeaker.tsx:43]: done

File: components/FaqDisclosure.tsx [hunk 5]
```diff
@@ -43,14 +43,31 @@ export default function FaqDisclosure({
...
           >
             <span>{question}</span>
-            <ChevronDownIcon className={`${isOpen ? 'transform rotate-180' : ''} w-5 h-5`} />
+            <ChevronDownIcon
+              className={`${
+                isOpen
+                  ? 'transform rotate-180 transition duration-500 ease-in-out'
+                  : 'transition duration-500 ease-in-out'
+              } w-5 h-5`}
+            />
```
[the remark this answers is not in the document]
Comment (thetechie7) [reply components/FaqDisclosure.tsx:52]: oops

**Review by MichaelZhao21 — APPROVED** [review]

**Review by NickBurnett — APPROVED** [review]

**Pull request merged**